src/app/_components/client/ConfirmModal/ConfirmModal.tsx

```diff
@@ -0,0 +1,67 @@
+import { Modal } from "@/app/_components/client";
+
+import Image, { StaticImageData } from "next/image";
+
+export interface ConfirmModalProps {
+  isShow: boolean;
+  title: string;
+  message?: string;
+  imageUrl?: StaticImageData;
+  completeText?: string;
+  incompleteText?: string;
+  handleClickClose: () => void;
+  onCompleteCheck: (isComplete: boolean) => void;
+}
+
+const ConfirmModal = ({
+  isShow,
+  title,
+  message,
+  imageUrl,
+  completeText = "확인",
+  incompleteText = "취소",
+  handleClickClose,
+  onCompleteCheck,
+}: ConfirmModalProps) => {
+  const handleClickComplete = () => {
+    onCompleteCheck(true);
+    handleClickClose();
+  };
+
+  const handleClickIncomplete = () => {
+    onCompleteCheck(false);
+    handleClickClose();
+  };
+
+  return (
+    <Modal isShow={isShow}>
+      <h3 className="mb-[2rem] pt-[2.4rem] text-size15 font-medium">{title}</h3>
+      {message && <p className="mb-[3.2rem] text-size13 font-regular">{message}</p>}
```

Comment:
[ P3 ]
메세지를 보여주는데는 문제없지만 사용하시는 분이 메세지를 줄바꿈을 하고 싶을때는 줄바꿈을 할 수 없는 문제점이 생길것같습니다!

따라서 


```suggestion
      {message &&
         <p 
            className="mb-[3.2rem] text-size13 font-regular" 
            dangerouslySetInnerHTML={{ __html: message }}
         />}
```

와같이 바꿔주시면 사용하시는분이 `<br>` 태그나 `<hr>`등 텍스트에 대해 자유롭게 커스텀하실 수 잇을것같아요

Reply:
좋은 제안 감사합니다 !! 
`dangerouslySetInnerHTML`로 HTML을 조작할 수 있는지는 처음 알았네요 😊

다만, XSS 공격에 좋지 않은 위험성이 높은 방식인 것 같아 내부 CSS를 통해 컨트롤하는 방식으로 수정해 봤습니다 !
외부에서 줄 바꿈 처리를 하고 싶다면 `{"안녕하세요\n 반갑습니다\n 안녕히 계세요"}` 변수로 text를 전달하면 됩니다 !

Reply:
옹 그래서 __html 이라는 키값을 주어 보안을 강화하는걸로 알고있는데 변경해주신 방식도 좋네요

Reply:
오오 `__html`이 보안을 통한 키값이었군요..?!
해당 속성에 대해서는 조금 더 찾아보면서 공부를 해보면 좋을 것 같네요 ㅎㅎ
감사합니다 !! 😀